### ISSUE TITLE
Offline hyperheuristics

### ISSUE THREAD

**Issue by minkull** [MitLware/MitLware-task-allocation#2]
Write a few paragraphs about existing literature on offline hyperheuristics. The aim of the literature review is for us to check what sort of techniques a new meta-heuristics framework should cover, including existing techniques and potentially future techniques.

Deadline: 31st August 2015.

**Comment by JJ** [comment]
You can assign yourself, or any other if you want.

**Comment by minkull** [comment]
In the menu on the right hand side, I can only assign myself, you, Jerry or Steven. However, I need to assign to another contributor. For example, one of these tasks is to be assigned to Emma Hart. However, I cannot make the assignment in the right hand side menu.

**Comment by JerrySwan** [comment]
JJ will be able to better advise on this than I, but I suspect it is because Emma has yet to accept her invitation.

**Comment by JJ** [comment]
you can name anyone with @whatsisname and it will arrive to the
corresponding inbox, even if he/she's not a repo contributor.

2015-08-11 22:20 GMT+02:00 JerrySwan <notifications@github.com>:

> JJ will be able to better advise on this than I, but I suspect it is
> because Emma has yet to accept her invitation.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MitLware/MitLware-task-allocation/issues/2#issuecomment-130053910>
> .
>



-- 
JJ

**Comment by minkull** [comment]
Hi JJ, sorry, I'm not very familiar with GitHub. When you say I can name anyone with @name, do you mean in that menu for assigning people on the right hand side? I've tried it in that menu, but it doesn't seem to work... I wonder if I should be entering the information somewhere else?

**Comment by JJ** [comment]
No, right here and now. If I say @fergunet, please say hi, or @JerrySwan
please say hi, or to someone outside the repo such as @pacastillo, please
say hi, they'll get an email notification.

2015-08-12 10:58 GMT+02:00 minkull <notifications@github.com>:

> Hi JJ, sorry, I'm not very familiar with GitHub. When you say I can name
> anyone with @name <https://github.com/name>, do you mean in that menu for
> assigning people on the right hand side? I've tried it in that menu, but it
> doesn't seem to work... I wonder if I should be entering the information
> somewhere else?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MitLware/MitLware-task-allocation/issues/2#issuecomment-130221923>
> .
>



-- 
JJ

**Comment by minkull** [comment]
Ohh, I see! I'll do that then in the other issue, because I still don't have the github id of Gisele, who will be responsible for this issue.

Thanks a lot for the clarification!

**Comment by minkull** [comment]
Hi @glpappa
